apps/mkt/components/automation-carousel/automation-benefit-icon.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  IconClock,
+  IconBug,
+  IconShield,
+  IconHammer,
+  IconAdjustments,
+} from "@tabler/icons-react";
+
+type IconType =
+  | "cycleTime"
+  | "failureRate"
+  | "compliance"
+  | "security"
+  | "techDebt"
+  | "prTitle";
+
+interface AutomationIconProps {
+  type: IconType;
+  color?: string;
+  stroke?: number;
+}
+
+const getIcon = (type: IconType) => {
+  switch (type) {
+    case "cycleTime":
+      return IconClock;
+    case "failureRate":
+      return IconBug;
+    case "compliance":
+      return IconAdjustments;
+    case "security":
+      return IconShield;
+    case "techDebt":
+      return IconHammer;
+    case "prTitle":
+      return IconHammer;
+    default:
+      throw new Error(`Unknown icon type: ${type}`);
+  }
+};
```

Comment:
This works well - I'll suggest to convert it to an object though, it's a bit less verbose :) 

You can do something like 
```ts
const icons: Record<IconType, ReactElement> = {
   cycleTime: IconClock,
   ...
}

return icons[type];
```